packages/web-frontend/cypress/scripts/generateConfig/VisualisationFilter.js

```diff
@@ -40,20 +48,25 @@ class VisualisationFilter {
       return objectUrls;
     }
 
+    const visualKey = this.getKey();
     const records = await this.fetchRecords(objectUrls);
-    const recordsById = keyBy(records, 'id');
+    // Camel case all the record keys
+    const transformedRecords = records.map(r => mapKeys(r, (_, key) => camel(key)));
```

Comment:
This is to ensure any snake cases will be converted to camel cases to match dashboardReportUrls files (eg: default.json). I think at the end, camel case is the standard convention for json keys naming

Reply:
Great call, kudos for paying attention to the details!